src/devices/Neo-M8/samples/NeoM8.Sample.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Iot.Device.Nmea0183;
+using Iot.Device.Nmea0183.Sentences;
+using Iot.Device.Gps;
+
+namespace Iot.Device.Gps.NeoM8Samples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (NeoM8 neoM8 = new NeoM8("/dev/ttyS0"))
+            {
+                bool gotRmc = false;
+                while (!gotRmc)
+                {
+                    TalkerSentence sentence = neoM8.Read();
```

Comment:
Can we also show an example of how you would use a QuerySentence?

Reply:
it's part of the spec, initially device didn't want to send anything (due to my fault, except I2C which I'm not sure why) and then I realized that you don't have to send a query to get anything from the device. I actually haven't tried it but left it since it was in the spec anyway and I've already tried if it constructs correct sentences

Reply:
mm I see, so neoM8 can only read but not request any data?